app/views/shared/svg/_icon_sprite.html.erb

```diff
@@ -75,5 +78,9 @@
     </g>
   </symbol>
 
+  <symbol id="svg-icon--tool" viewBox="0 0 331.6 308.66">
```

Comment:
This already exists in master. If we rebase this branch (and also `global-nav-master` from latest `master` we will get this for free. If we leave it in, it's going to cause a merge conflict, I think.